core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/user/workflow/WorkflowExecutionsResource.scala

```diff
@@ -29,35 +29,15 @@ object WorkflowExecutionsResource {
     executionsDao.fetchOneByEid(eId)
   }
 
-  def getExpiredResults(timeToLive: Int): List[ExecutionResultEntry] = {
+  def getExpiredExecutionsWithResultOrLog(timeToLive: Int): List[WorkflowExecutions] = {
```

Comment:
the name `WorkflowExecutions` indicates it contains multiple workflow execution instances. why wrapping it with another List? is WorkflowExecutions referring to one execution?

Reply:
We have a class called `WorkflowExecutions`. One `WorkflowExecutions` refers to one execution. Although I feel this name is weird but changing it is out of scope of this PR.